global/meta/models.yml

```diff
@@ -3846,6 +3846,10 @@ import_preview:
   name:
     type: string
     required: true
+    enum:
```

Comment:
Please add also motion and committee

Reply:
This should be done in the relevant PRs - the current state of the main does not support motion and committee imports.

Reply:
okay